Dockerfile

```diff
@@ -14,7 +14,12 @@ RUN echo http://nl.alpinelinux.org/alpine/v3.6/main > /etc/apk/repositories; \
 RUN apk update && apk add make && apk add bash
 RUN apk add curl && apk add git && apk add openssh
```

Comment:
We could remove your script from below and add `glide` to the `apk add` command here.

Reply:
Analogous to what happens here in config-seed Dockerfile https://github.com/edgexfoundry/edgex-go/blob/master/docker/Dockerfile.config-seed#L28-L34.
The issue you are currently hitting after you download the binary is that it's not on the PATH.
Installing via apk will solve this.

Reply:
@jamesrgregg disregard everything I just said above, my apologies.  I just realized you said there was a compilation issue and I see it now.

Reply:
I also reverted back to the previous build image.  The error is indicating that gcc isn't in the path.
I can perhaps try to add gcc via apk.